src/node_sqlite.h

```diff
@@ -1,6 +1,7 @@
 #ifndef SRC_NODE_SQLITE_H_
 #define SRC_NODE_SQLITE_H_
 
+#include <string_view>
```

Comment:
Can you move this after line 7. We need to include dependencies after NODE_WANT_INTERNALS